docs/topics/data-analysis/data-science-libraries.md

```diff
@@ -1,88 +1,40 @@
+| **Library**                                                                | **Purpose**                                                                                           | **Features**                                                                                                                                                                                                                                                                                                                                                                                                    |
+|------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Tablesaw](https://github.com/jtablesaw/tablesaw)                      | - Data collection <br/> - Data cleaning and processing <br/> - Data exploration and visualization | - Tools for loading, cleaning, transforming, filtering, and summarizing data <br/> -  Inspired by [Plot.ly](https://plotly.com/)                                                                                                                                                                                                                                                                            |
+| [CoreNLP](https://stanfordnlp.github.io/CoreNLP/)                      | Data cleaning and processing                                                                      | - Natural language processing toolkit <br/> - Linguistic annotations for text, such as sentiment and quote attributions <br/> - Support for eight languages                                                                                                                                                                                                                                                 |
+| [Smile](https://github.com/haifengl/smile)                             | - Data cleaning and processing <br/> - Data exploration and visualization <br/> - Model building  | - Ready-made algorithms for machine learning and natural language processing <br/> - Linear algebra, graph, interpolation, and visualization tools <br/> - Smile provides a functional [Kotlin API](https://github.com/haifengl/smile/tree/master/kotlin), [Scala API](https://github.com/haifengl/smile/tree/master/scala), [Clojure API](https://github.com/haifengl/smile/tree/master/clojure), and more |
```

Comment:
We never repeat the tool's name in the descriptions for other tools, so I suggest omitting it here as well. (i.e.: Smile provides --> provides)